algorithms/kernel/dbscan/oneapi/dbscan_kernel_ucapi.h

```diff
@@ -82,14 +83,16 @@ class DBSCANBatchKernelUCAPI : public Kernel
 
     uint32_t getWorkgroupNumber(uint32_t numberOfChunks) { return numberOfChunks * _minSubgroupSize / _maxWorkgroupSize + 1; }
 
-    services::Status initializeBuffers(uint32_t nRows);
+    services::Status initializeBuffers(uint32_t nRows, uint32_t chunkSize);
 
     services::Status buildProgram(oneapi::internal::ClKernelFactoryIface & kernel_factory);
 
-    static const size_t _minSubgroupSize  = 16;
-    static const size_t _maxWorkgroupSize = 256;
-    static const size_t _chunkNumber      = 64;
-    static const size_t _queueBlockSize   = 64;
+    static const size_t _minSubgroupSize              = 16;
+    static const size_t _maxWorkgroupSize             = 256;
+    static const size_t _minRecommendedNumberOfChunks = 64;
```

Comment:
```suggestion
    static const size_t _minNumberOfChunks = 64;
```
probably will be better this way?

Reply:
It might be confusing because real value can be less than _minNumberOfChunks in that case